Themes/default/languages/index.english.php

```diff
@@ -480,7 +480,7 @@
 
 $txt['page_created_full'] = 'Page created in %1$.3f seconds with %2$d queries.';
 
-$txt['report_to_mod_func'] = 'Use this function to inform the moderators and administrators of an abusive or wrongly posted message.';
+$txt['report_to_mod_func'] = 'Use this function to inform the moderators and administrators of an abusive or message posted in error.';
```

Comment:
The old version was awkward, but this is grammatically broken. How about "an abusive message or one posted in error," or more simply "a problematic message"?